### PR TITLE
_machinery.dm cleanup

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
Code cleanup, update to latest contribution guidelines

### CODE REVIEW

File: code/game/machinery/_machinery.dm [hunk 1]
```diff
@@ -820,59 +827,69 @@
...
+		return FALSE
+
+	var/obj/item/circuitboard/machine/machine_board = locate(/obj/item/circuitboard/machine) in component_parts
+	var/required_type
+	if(replacer_tool.works_from_distance)
+		to_chat(user, display_parts(user))
+	if(!machine_board)
+		return FALSE
+
+	for(var/obj/item/primary_part in component_parts)
```
Comment (Watermelon914) [inline code/game/machinery/_machinery.dm:854]: is component_parts a list that can only contain items?

Comment (Ghilker) [reply code/game/machinery/_machinery.dm:854]: In theory yes, but I'm not sure

Comment (Watermelon914) [reply code/game/machinery/_machinery.dm:854]: you should probably do `as anything` in that case so that we can catch out the cases they are not.

Comment (Ghilker) [inline code/game/machinery/_machinery.dm:854]:
```suggestion
	for(var/obj/item/primary_part as anything in component_parts)
```

Comment (Ghilker) [reply code/game/machinery/_machinery.dm]: done

**Pull request merged**